msteams-platform/messaging-extensions/build-api-based-message-extension.md

```diff
@@ -0,0 +1,462 @@
+---
+title: Create API based Message extension
+author: v-ypalikila
+description: Learn how to build an API message extension and API plugin using Teams developer portal and Teams Toolkit.
+ms.localizationpriority: medium
+ms.topic: overview
+ms.author: anclear
+ms.date: 09/07/2023
+---
+
+# API based Message extension
+
+> [!NOTE]
+> API based message extension only supports search commands.
+
+API based message extensions use a web service to handle user requests and responses. They don't require a bot registration or a bot framework SDK. They can be configured and deployed using the Developer Portal for Teams or the Teams Toolkit.
+
+You can create an API based message extension in Teams using an OpenAPI Specification document. After you've created an OpenAPI Specification document for the APIs you want to use, upload the OpenAPI Specification document to Teams Toolkit or Developer portal for Teams to generate and integrate the client code in your app's project. Create or generate an Adaptive Card template to handle the responses from the API.
+
+An API based message extension helps to interact directly with third-party data, apps, and services, enhancing its capabilities and broadening its range of capabilities. With APIs for message extension, you can:
```

Comment:
How about:

API-based message extensions enable your Teams app to interact directly with third-party data, apps, and services. With API-based message extensions, you can:

Reply:
API based message extensions help your apps to interact directly with third-party data, apps, and services, enhancing its capabilities. With APIs for message extension, you can: